tests/filter_plugins

```diff
@@ -0,0 +1 @@
+../provision/filter_plugins
```

Comment:
@herlo : tried adding it to sys.path . I am afraid , its not working because of the same reason i mentioned above . Apperently , ansible searches for filter_plugins from point of invocation . 
we either have to maintain a symlink or , add it to ansible.cfg .